workers/unity/Assets/Playground/Scripts/Worker/MobileClientWorkerConnector.cs

```diff
@@ -1,20 +1,49 @@
+using System;
+using System.Net;
 using Improbable.Gdk.Core;
 using Improbable.Gdk.Mobile;
+using Improbable.Worker.CInterop;
 using UnityEngine;
 
 namespace Playground
 {
-    public class MobileClientWorkerConnector : DefaultMobileWorkerConnector
+    public class MobileClientWorkerConnector : WorkerConnector, IMobileSettingsProvider
     {
 #pragma warning disable 649
         [SerializeField] private GameObject level;
+        [SerializeField] private string ipAddress;
 #pragma warning restore 649
 
         private GameObject levelInstance;
 
         public async void Start()
         {
-            await Connect(WorkerUtils.MobileClient, new ForwardingDispatcher()).ConfigureAwait(false);
+            var connParams = new ConnectionParameters
+            {
+                WorkerType = WorkerUtils.MobileClient,
+                DefaultComponentVtable = new ComponentVtable()
+            };
+
+            var flowSettingsProvider = new MobileConnectionFlowModifier(new PlayerPrefsSettingsProvider(), new CommandLineSettingsProvider(), this);
```

Comment:
I think the ordering needs to be:
1. first check if we got new command line args
2. then check if player prefs available
3. then do whatever fallback behaviour you have

cause otherwise command line args will be ignored after player prefs have been set